package.json

```diff
@@ -70,11 +70,17 @@
     "deploy-configure": "hardhat deploy --network",
     "docs": "hardhat docgen",
     "format": "npm run format:sol && npm run format:ts",
-    "format:sol": "prettier -c -w --no-editorconfig contracts/**/*.sol contracts/**/**/*.sol",
-    "format:ts": "prettier -c -w --no-editorconfig **/*.ts",
+    "format:fix": "npm run format -- --write",
```

Comment:
I don't think this works as expected because the `--write` will be used in the second part of the script only:
```
npm run format:sol && npm run format:ts -- write
```

Reply:
let me check it, I tried but I did not realize if it was applied to both

Reply:
updated